### ISSUE TITLE
TypeError in makeARGB

### ISSUE THREAD

**Issue by 2xB** [pyqtgraph/pyqtgraph#872]
From executing pyqtgraph/graphicsItems/tests/test_ImageItem.py::test_ImageItem with Python 2.7 (Anaconda):

```
Traceback (most recent call last):
  File "/home/travis/repo-clone/pyqtgraph/graphicsItems/ImageItem.py", line 435, in paint
    self.render()
  File "/home/travis/repo-clone/pyqtgraph/graphicsItems/ImageItem.py", line 427, in render
    argb, alpha = fn.makeARGB(image, lut=lut, levels=levels)
  File "/home/travis/repo-clone/pyqtgraph/functions.py", line 1107, in makeARGB
    data = rescaleData(data, scale/(maxVal-minVal), minVal, dtype=dtype)
TypeError: numpy boolean subtract, the `-` operator, is deprecated, use the bitwise_xor, the `^` operator, or the logical_xor function instead.
```

**Comment by j9ac9k** [comment]
This is an issue on Python 3 as well, it has to do with maxVal and minVal accepting booleans.

**Comment by Dschoni** [comment]
Closed by #876
